pkg/console/dashboard_panels.go

```diff
@@ -32,6 +33,9 @@ const (
 	statusSettingUpNode = "Setting up node"
 	statusSettingUpHarv = "Setting up Harvester"
 
+	defaultHarvesterConfig = "/oem/harvester.config"
+	defaultCustomConfig    = "/oem/99_custom.yaml"
```

Comment:
Should be `90_custom.yaml`?
![](https://github.com/harvester/harvester-installer/assets/15064560/a4c8c118-af42-47a8-a925-ee58bfc5b50e)

```bash
harvester-bl79m:/oem # ls
90_custom.yaml  elemental.config  grubenv  harvester.config  install  lost+found
harvester-bl79m:/oem #
```

Reply:
This PR didn't have this problem last week

Reply:
i think i renamed it incorrectly.

Reply:
when i moved the hardcoded file names in the custom copy directives, i have accidentally changed the order.